graphql/resolvers.go

```diff
@@ -2,22 +2,73 @@ package graphql
 
 import (
 	"context"
+	"time"
+
+	"github.com/evergreen-ci/evergreen/model/patch"
+	"github.com/evergreen-ci/evergreen/rest/data"
+	"github.com/pkg/errors"
 )
 
-type Resolver struct{}
+type Resolver struct {
+	sc data.Connector
```

Comment:
I don't see how this is used.

Reply:
https://github.com/evergreen-ci/evergreen/pull/3013/files#diff-38da1fe0dd4e3c4980176c1833b102e8R53